qt/applications/workbench/workbench/plugins/editor.py

```diff
@@ -61,10 +61,16 @@ def __init__(self, parent):
 
         # attributes
         self.tabs_open_on_closing = None
+        self.run_action = create_action(self, "Run",
```

Comment:
It looks to me as if we have a copy-and-paste error here as we have the same code twice?

Reply:
I believe this is a merge conflict I did incorrectly let me fix this.